lib/repl.js

```diff
@@ -1067,7 +1067,7 @@ REPLServer.prototype.memory = function memory(cmd) {
           self.lines.level.push({
             line: self.lines.length - 1,
             depth: depth,
```

Comment:
It seems to be the object of another PR. But I recall that some collaborators oppose partial shorthands: this is preferred to be consistent, all shorthands or all full pairs.